website/docs/r/s3tables_namespace.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "S3 Tables"
+layout: "aws"
+page_title: "AWS: aws_s3tables_namespace"
+description: |-
+  Terraform resource for managing an AWS S3 Tables Namespace.
```

Comment:
The branding is _Amazon S3 Tables_, not _AWS S3 Tables_.

Reply:
Good catch